lib/graphiti/sideload/polymorphic_belongs_to.rb

```diff
@@ -15,9 +15,14 @@ def method_missing(name, *args, &blk)
   class Grouper
     attr_reader :field_name
 
-    def initialize(field_name)
+    def initialize(field_name, opts={})
       @field_name = field_name
       @groups = []
+      @ignore_groups = Array(opts[:ignore_groups])
```

Comment:
This looks implemented correctly, but I'd prefer `ignore` over `ignore_groups`

Reply:
Tend to agree with the above comment around `except` instead of `ignore`, since that matches our options elsewhere in the library/rails defaults.

Reply:
Changed.  Please let me know what you think.